connectors/src/connectors/zendesk/index.ts

```diff
@@ -99,7 +99,21 @@ export class ZendeskConnectorManager extends BaseConnectorManager<null> {
   }
 
   async clean(): Promise<Result<undefined, Error>> {
-    throw new Error("Method not implemented.");
+    const { connectorId } = this;
+    const connector = await ConnectorResource.fetchById(connectorId);
+    if (!connector) {
+      logger.error({ connectorId }, "[Zendesk] Connector not found.");
+      return new Err(new Error("Connector not found"));
+    }
+
+    const result = await connector.delete();
```

Comment:
Does that automatically delete all related datasources ?

Reply:
After looking at the code of google drive, it seems that you need to implements more in `ZendeskConnectorStrategy` (maybe it's planned in another PR?)

Reply:
No, this is handled by front; when it calls the endpoint `connectors/delete/:connector_id` it also calls `coreAPI.deleteDataSource`

Reply:
yes this is planned in another PR 👍

Reply:
done! (implemented the deletion of the resources associated to a connector in `ZendeskConnectorStrategy.delete`).